xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/Logic.js

```diff
@@ -621,6 +680,23 @@ define([
      */
 
 
+    /**
+     * Return whether selecting properties is enabled
+     * If entry is given, return whether this entry can be selected
+     * @param {Object} [parameters]
+     * @param {Object} [parameters.entry]
+     */
+    isSelectionEnabled ({ entry } = {}) {
+      // TODO: Ensure entry is valid (need other current PR)
+      if (entry !== undefined) {
+        return entry.age < 40;
+        // return this.isEditable({ entry });
```

Comment:
You should check if the entry has a value set for the configured id property (from the entry descriptor). We can't select entries that don't have an id. Then, if the id is specified we look at the configuration (TODO).